studio-client/apps/main/content-hub-adapter-rss/package.json

```diff
@@ -11,12 +11,12 @@
     "@jangaroo/runtime": "^1.0.0"
   },
   "devDependencies": {
-    "@jangaroo/build": "^1.0.0",
-    "@jangaroo/core": "^1.0.0",
-    "@jangaroo/publish": "^1.0.0",
-    "@jangaroo/run": "^1.0.0",
-    "eslint": "7.27.0",
-    "rimraf": "^3.0.2"
+    "@jangaroo/build": "^1.5.0",
+    "@jangaroo/core": "^1.5.0",
+    "@jangaroo/publish": "^1.5.0",
+    "@jangaroo/run": "^1.5.0",
```

Comment:
@aholtkamp While discussing updating cm versions in plugins we came across node, pnpm and jangaroo versions. We want to stay compatible with old cms releases as long as possible and avoid cms release specific maintenance branches.

Quest now is, if the version updates in the package.json are reasonable for you? We left the jangaroo/runtime version unchanged to stay compatible with older studio installations. All the other dependencies should be used to build the plugin artefact internally.

Reply:
For runtime only the normal "dependencies" are relevant. So this change looks good to me if we want to avoid having a maintenance branch while still being compatible to older cms releases.